core/common/src/main/java/alluxio/client/file/URIStatus.java

```diff
@@ -40,14 +40,17 @@
 @ThreadSafe
 public class URIStatus {
   private final FileInfo mInfo;
+  /** Context related to Presto runtime associated with this URI. */
+  private PrestoContext mPrestoContext;
```

Comment:
How do we distinguish the responsibilities of info and context?  especially the two fields in prestoContext looks generic.  I might say the quota and scope should be under mInfo rather than mPrestoContext.

Reply:
`FileInfo` is coming from Alluxio which doesn't have such concepts.

Reply:
Actually, I would like to consolidate `FileInfo` with `URIStatus` too. There are historical reasons these are two different classes.